hw/ip/uart/doc/dv_plan/index.md

```diff
@@ -28,14 +28,14 @@ UART testbench has been constructed based on the
 ### Top level testbench
 Top level testbench is located at `hw/ip/uart/dv/tb/tb.sv`. It instantiates the UART DUT module `hw/ip/uart/rtl/uart.sv`.
 In addition, it instantiates the following interfaces, connects them to the DUT and sets their handle into `uvm_config_db`:
-* [Clock and reset interface]({{< relref "hw/dv/sv/common_ifs/README.md" >}})
+* [Clock and reset interface]({{< relref "hw/dv/sv/common_ifs/index.md" >}})
```

Comment:
Also update `util/uvmdvgen/dv_plan.md.tpl`

Reply:
Oh, I forgot to update template. Please find updated file in the latest commit.